### PR TITLE
Hotfix 1.28.2 - Migrate to the high-availability RPC for Gnosis chain

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

The old (https://rpc.gnosischain.com/) Gnosis chain RPC endpoint is not stable and gets down sometimes.
Corresponding to https://docs.gnosischain.com/tools/rpc/#gateway we should https://rpc.gnosis.gateway.fm endpoint, because it provides a high-availability public RPC as part of their Core Contributor agreement in [GIP-70].

### CODE REVIEW

File: src/custom/utils/switchChain.ts [hunk 1]
```diff
@@ -16,24 +16,6 @@ function getRpcUrls(chainId: SupportedChainId): [string] {
...
-    // case SupportedChainId.POLYGON:
-    //   return ['https://polygon-rpc.com/']
-    // case SupportedChainId.POLYGON_MUMBAI:
-    //   return ['https://rpc-endpoints.superfluid.dev/mumbai']
-    // case SupportedChainId.CELO:
-    //   return ['https://forno.celo.org']
-    // case SupportedChainId.CELO_ALFAJORES:
-    //   return ['https://alfajores-forno.celo-testnet.org']
     case SupportedChainId.GNOSIS_CHAIN:
       return ['https://rpc.gnosischain.com/']
```
Comment (alfetopito) [inline src/custom/utils/switchChain.ts:20]: Why isn't this changed?

**Review by alfetopito — APPROVED** [review]
Remember to NOT squash!

**Review by elena-zh — APPROVED** [review]
Changes LGTM!
However, I'd propose to add a warning in GC to change RPC in MM for user to make it work nicely there https://cowservices.slack.com/archives/C0361CDG8GP/p1673432270797119

Besides, it would be nice to add https://rpc.gnosis.gateway.fm/ PRC into the adding network request to MM (if it is possible, of cause)
image

File: src/custom/utils/switchChain.ts (hunk 1 above)
Comment (alfetopito) [reply src/custom/utils/switchChain.ts:20]: @shoom3301 ?

**Pull request merged**